app/scripts/controllers/build.js

```diff
@@ -63,7 +63,7 @@ angular.module('openshiftConsole')
     var watches = [];
 
     var setLogVars = function(build) {
-      $scope.logCanRun = !(_.includes(['New', 'Pending', 'Error'], build.status.phase));
+      $scope.logCanRun = !(_.includes(['New', 'Pending', 'Error'], _.get(build, 'status.phase')));
```

Comment:
I'm confused how `buildResolved` is called with a null value? Just worried the problem runs deeper.

Reply:
I noticed it due to a timeout with an API response (I think).

Reply:
Shouldn't that have gone to the `buildRejected` callback?

Reply:
That would make sense.   Hmm. Not sure the cause then.